src/Objects.ts

```diff
@@ -0,0 +1,13 @@
+export type Item = {
+  id: string,
+  type?: ItemType,
+  content?: string,
+}
+
+export enum ItemType {Todo, Done}
+
+export type Action = {type: ActionType, payload: any}
```

Comment:
액션은 정확하게 작성해주시는게 좋습니다. 각 action별로 payload의 타입이 전부 다를텐데, 이걸 컴파일 타임에 검사를 안해주면 런타임에 잘못된 코드가 들어왔을 때 대비가 안될 수 있죠.

Reply:
맞습니다, 제가 any로 두고 고치질 않았군요. 수정하겠습니다 :)
```suggestion
export type Action = {type: ActionType, payload: Item}
```